content/en/docs/How-to guides/clone-repository.md

```diff
@@ -0,0 +1,498 @@
+		[fetch-source : clone] {"level":"info","ts":1652727926.1650486,"caller":"git/git.go:170","msg":"Successfully cloned https://github.com/tektoncd/website @ 537ee0903a3874f5ddf4148049b0e8563e4338f9 (grafted, HEAD) in path /workspace/output/"}
+		[fetch-source : clone] {"level":"info","ts":1652727926.1901643,"caller":"git/git.go:208","msg":"Successfully initialized and updated submodules in path /workspace/output/"}
+		[fetch-source : clone] + cd /workspace/output/
+		[fetch-source : clone] + git rev-parse HEAD
+		[fetch-source : clone] + RESULT_SHA=537ee0903a3874f5ddf4148049b0e8563e4338f9
+		[fetch-source : clone] + EXIT_CODE=0
+		[fetch-source : clone] + '[' 0 '!=' 0 ]
+		[fetch-source : clone] + printf '%s' 537ee0903a3874f5ddf4148049b0e8563e4338f9
+		[fetch-source : clone] + printf '%s' https://github.com/tektoncd/website
+
+### Git authentication
+
+To run the previous example on a private repository, you must create a [Kubernetes
+Secret][secrets] with your credentials, then pass that secret to your Task as a
+Workspace.
+
+The following steps show you how to authenticate using an ssh key. For other
+forms of authentication check the [git-clone Task documentation][git-clone].
+
+1.  Create a Kubernetes Secret with your credentials, for example:
+     
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: git-credentials
+    data:
+      id_rsa: AS0tLS...
+      known_hosts: AG033S...
+      config: GS0FFL...
+    ``` 
+
+    The values for the fields under `data` are the corresponding base64-encoded
+    files in the `.ssh` directory. For example, for `id_rsa` copy-paste the
+    output of:
+
+    ```bash
+    cat ~/.ssh/id_rsa | base64 -w0
+    ```
+
+1.	Update `pipeline.yaml`, add a new Workspace to both the Pipeline and the
+    Task:
+
+    ```yaml
+    workspaces:
+    - name: shared-data
+      description: | 
+        This workspace contains the cloned repo files, so they can be read by the
+        next task.
+    - name: git-credentials
+      description: My ssh credentials
+    tasks:
+    - name: fetch-repo
+      taskRef:
+        name: git-clone
+      workspaces:
+      - name: output
+        workspace: shared-data
+      - name: ssh-directory
+        workspace: git-credentials
+    ```
+
+    The Workspace `git-credentials` is defined at Pipeline level and then passed to
+    to the Task with the name `ssh-directory`, which is the name the Task
+    expects.
+
+
+1.  Update `pipelinerun.yaml` to use the Secret as a Workspace and change the
+    git URL from https to ssh:
+
+    ```yaml
+    workspaces:
+    - name: shared-data
+      volumeClaimTemplate:
+        spec:
+          accessModes:
+          - ReadWriteOnce
+          resources:
+            requests:
+              storage: 1Gi
+    - name: git-credentials
+      secret:
+        secretName: git-credentials
+		params:
+		- name: repo-url
+			value: git@github.com:tektoncd/website.git
```

Comment:
should be spaces instead of tabs

Reply:
Good catch! The pitfalls of copy-pasting stuff around :(

I should make a habit of using `:retab` with every code sample.